ServiceNow Service Catalog v2.5.0 updates, Washington DC updates. (#567)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/developer-reference.mdx

| Create Workspace              | Creates a new HCP Terraform workspace from VCS repository.                                                                                                                 |
| Create Workspace with Vars    | Creates a new HCP Terraform workspace from VCS repository and creates any variables provided.                                                                              |
| Create Run                    | Creates/queues a new run in the HCP Terraform workspace.                                                                                                                   |
| Apply Run                     | Applies a run in the HCP Terraform workspace.                                                                                                                              |
| Provision Resources           | Creates a new HCP Terraform workspace (with auto-apply), creates/queues a run, applies the run when ready.                                                                     |

> | Provision Resources with Vars | Creates a new HCP Terraform workspace (with auto-apply), creates any variables, creates/queues a run, applies the run when ready.  

Author: Unfortunately, that's the name of the workflow in ServiceNow!

> | Provision Resources with Vars | Creates a new HCP Terraform workspace (with auto-apply), creates any variables, creates/queues a run, applies the run when ready.  

Reviewer: Ah, gotcha! I had a feeling, thank you for confirming though! Updated my suggestion to match!

| Provision No-Code Workspace and Deploy Resources | Creates a new HCP Terraform workspace based on a no-code module configured in the private registry (with auto-apply), creates any variables, creates/queues a run, applies the run when ready.                                                |
| Delete Workspace              | Creates a destroy run plan.                                                                                                                                                  |
| Worker Poll Run State         | Polls the HCP Terraform API for the current run state of a workspace.                                                                                                      |
| Worker Poll Apply Run         | Polls the HCP Terraform API and applies any pending Terraform runs.                                                                                                        |
| Worker Poll Destroy Workspace | Queries ServiceNow Terraform Records for resources marked `is_destroyable`, applies the destroy run to destroy resources, and deletes the corresponding Terraform workspace. |

> | Update No-Code Workspace and Deploy Resources | Updates a previously created no-code workspace to new no-code provisioning settings when a new release of the no-code module is available. Updates variable values attached to the workspace and starts a new Terraform run.     

Reviewer: ```suggestion
| Update No-Code Workspace and Deploy Resources | Updates an existing no-code workspace to the most recent no-code module version, updates that workspace's attached variable values, and then starts a new Terraform run.     
```
